src/js/modules/infragistics.ui.layoutmanager.js

```diff
@@ -311,31 +312,78 @@
                     // Set
                     $('.selector').igLayoutManager('option', 'gridLayout', { columnWidth: 400, columnHeight: 500});
                     ```
-                    string The column height can be set in pixels (px), %, em and other units.
+                    string The column height can be set in pixels (px), %, em or *.
                     number The column height can be set as a number.
                     */
                     columnHeight: null,
-                    /* type="string|number" Accepts number or string with width in px or percents
+                    /* type="array" Gets/sets a list of column heights. Each height could be a number,
+                        string with height in px, percents
+                        or asterisk (*) which will fill the remaining height.
+                        If more than one column has an asterisk value, the remaining height will
+                        be equally distributed between the columns.
                     ```
-                    //Initialize
+                    // Initialize
                     $('.selector').igLayoutManager({
                         layoutMode: "grid",
                         gridLayout: {
-                        columnWidth: 200
+                            columnsHeight: ["200px", "10%", "*", "*"]
                         }
                     });
 
-                    //Get
+                    // Get
+                    var gridLayout = $('.selector').igLayoutManager('option', 'gridLayout');
+                    var columnsHeight = gridLayout.columnsHeight;
+
+                    // Set
+                    $('.selector').igLayoutManager('option', 'gridLayout', { columnsHeight: ["200px", "10%", "*", "250px"]});
+                    ```
+                    */
+                    columnsHeight: [],
```

Comment:
Same comment as in the tile manager. You should make the change on the `columnHeight` option, rather than introduce a new option.